src/components/CmsZone.vue

```diff
@@ -224,6 +241,29 @@ export default class CmsZone extends Vue {
     return (index: number) => `cms-zone-content-${this.zoneId}-${index}`;
   }
 
+  get zoneType(): string {
+    if (!this.lastResponse || this.zoneStatus === 'error') {
+      return '';
+    }
+    return this.lastResponse.zone_type;
+  }
+
+  get zoneHeader(): string {
+    return this.lastResponse && this.lastResponse.zone_header
+      ? `<div class="zone-header">${this.lastResponse.zone_header}</div>`
+      : '';
+  }
+
+  get zoneFooter(): string {
+    return this.lastResponse && this.lastResponse.zone_footer
```

Comment:
nit: similarly to above `this.lastResponse?.zone_footer`

Reply:
:+1: